train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -51,4 +57,15 @@ public void setJoystickPosition(int joystickPosition) {
 	public Tachograph getTg(){
 		return tg;
 	}
+
+	public void startSimulation(){
+		if(timer!=null)return;
+		timer=new Timer();
+		timer.schedule(new TimerTask() {
+			@Override
+			public void run() {
+				followSpeed();
+			}
+		},50);
+	}
```

Comment:
I think this time is too low

Reply:
I don't think so, but I change it anyways. Thank you for your reviews.